recipe/meta.yaml

```diff
@@ -88,7 +88,7 @@ outputs:
         # CuPy default detects CUDA from nvcc, but on Conda-Forge's dockers nvcc lives in a different place...
         # With conda-forge/nvcc-feedstock#58, CUDA_PATH is set correctly
         - export NVCC="$(which nvcc)"                                                # [linux]
-        - export CUPY_NUM_BUILD_JOBS=1                                               # [aarch64 or ppc64le or (linux64 and cuda_compiler == "nvcc")]
+        - export CUPY_NUM_BUILD_JOBS=1
```

Comment:
Looks like Linux x86_64 CUDA 12.8 is doing better, which was the issue before

Windows is still seeing memory warnings